app/main.py

```diff
@@ -1,8 +1,27 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = []
+
+    # Create Person instances
+    for person_info in people:
+        person = Person(person_info["name"], person_info["age"])
+        person_list.append(person)
+
+    # Set spouse relationships
+    for person_info in people:
+        person = Person.people[person_info["name"]]
+        if "wife" in person_info and person_info["wife"] is not None:
+            person.wife = Person.people[person_info["wife"]]
+        elif "husband" in person_info and person_info["husband"] is not None:
+            person.husband = Person.people[person_info["husband"]]
```

Comment:
The code attempts to set both `wife` and `husband` attributes on the `Person` instances, but these attributes are not defined in the `Person` class. This will result in an `AttributeError` when trying to access these attributes. Consider defining these attributes in the `Person` class, possibly initializing them to `None` in the constructor.